docs/source/blog/derotation.md

```diff
@@ -0,0 +1,83 @@
+:blogpost: true
+:date: July 1, 2025
+:author: Laura Porta
+:location: London, UK
+:category: Blog
+:language: English
+:image: 1
+
+# `derotation`: a Python package for correcting motion artifacts in rotating multiphoton movies
+
+The `derotation` package provides a robust solution for reconstructing multiphoton movies of rotating samples acquired with a line-scanning microscope at high speeds and low frame rates. When imaging a rotating sample, the line-by-line nature of acquisition introduces significant geometric distortions. This package corrects these artifacts to produce clear, stable movies suitable for standard analysis pipelines like Suite2p.
```

Comment:
At first I wasn't sure if "at high speed" was referring to high rotation speed, line scanning speed, acquisition speed, sth else?

I think you mean rotation speed but better make it explicit.

Reply:
oh it's rotation speed 😂 I will make it more specific